src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -464,9 +466,22 @@ protected override void Dispose(bool disposing)
         {
```

Comment:
We need to do two things:
1. We need a finalizer
2. call the base.Dispose(disposing); <--- There is disagreement on what to pass according to: https://msdn.microsoft.com/en-us/library/fs2xkftw(v=vs.110).aspx